internal/testutil/testutil.go

```diff
@@ -0,0 +1,42 @@
+package testutil
+
+import (
+	"embed"
+)
+
+//go:embed data
+var testDataDir embed.FS
+
+// Name of the root folder in embed.FS
+const testDataRootDir = "data"
+
+// Version represents a kubernetes version. Mock servers can be instantiated to return known output for a given version.
+type Version string
+
+// Server returns an HTTP Server for the given version, ready to serve static endpoints for KSM, Kubelet and CP components.
+func (v Version) Server() (*Server, error) {
+	return newServer(v)
+}
+
+// List of all the versions we have testdata for.
+// When adding a new version:
+// - REMEMBER TO ADD IT TO AllVersions() BELOW.
+// - UPDATE LatestVersion() BELOW IF NEEDED
+
+const (
+	Testdata116 = "1_16"
+	Testdata118 = "1_18"
+)
+
+// LatestVersion returns the latest version we have test data for.
+func LatestVersion() Version {
+	return Testdata118
```

Comment:
Would make sense instead of returning a static field we need to update to order Version with sort and return the last element?

https://play.golang.org/p/Hy2x-W0eBT0

Reply:
Not sure about the `sort` part (as I was thinking of the possibility of having "weirder" versions int he future), but you're totally right with using the last element of AllVersions.

Reply:
Done! We still rely on the user for the sorting, as it could lead to weird effects if we start adding strange stuff, like `1_18KSM2_0` or the likes of that.